source/language/types.rst

```diff
@@ -226,9 +226,9 @@ namespace are listed in table `1 <#tab:real-constants>`__.
       +-------------------------------+--------------+--------------+---------------------+
       | Constant                      | Alphanumeric | Unicode      | Approximate Base 10 |
       +-------------------------------+--------------+--------------+---------------------+
-      | (r)1-1(lr)2-2(rl)3-3(l)4-4 Pi | pi           | :math:`\pi`  | 3.1415926535...     |
+      | Pi                            | pi           | π            | 3.1415926535...     |
       +-------------------------------+--------------+--------------+---------------------+
-      | Tau                           | tau          | :math:`\tau` | 6.283185...         |
+      | Tau                           | tau          | τ            | 6.283185...         |
       +-------------------------------+--------------+--------------+---------------------+
       | Euler’s number                | euler_gamma  | :math:`e`    | 2.7182818284...     |
```

Comment:
I think this is incorrect and the Euler gamma is Euler's constant rather than Euler's number. This should probably just be `euler`. I am not sure if there is a good unicode character for Euler's number other than https://www.compart.com/en/unicode/U+0065.

Reply:
Fair point, but I wonder if we should actually drop this one from the table and instead just add `exp(...)`` to the built-in scientific calculator functions?

Reply:
👍  to that.

Reply:
@taalexander @blakejohnson maybe we can open a follow up for reviewing symbols, in general, trying to use more semantic ones, if possible.